standalone/pruntime/enclave/src/system/gk.rs

```diff
@@ -223,7 +240,10 @@ where
     }
 
     pub fn push_gatekeeper_message(&self, message: impl Encode + BindTopic) {
-        self.egress.push_message(message)
+        self.egress
+            .as_ref()
+            .expect("gk should work after acquiring master key; qed.")
```

Comment:
Consider to make it a function, because this `expect` is used everywehre

Reply:
fixed